Sources/VideoOptionsControlBar.swift

```diff
@@ -24,66 +24,110 @@ protocol VideoOptionsControlBarDelegate: class {
+        rptButton.setImage(UIImage(named: "iconNoRepeat"), for: .normal)
+        rptButton.tintColor = .white
+        rptButton.accessibilityHint = NSLocalizedString("REPEAT_MODE", comment: "")
+        rptButton.accessibilityLabel = NSLocalizedString("REPEAT_MODE_HINT", comment: "")
         return rptButton
     }()
     
-    lazy var orientationLockButton: UIButton = {
-        var orientLockButton = UIButton(type: .system)
-        orientLockButton.addTarget(self, action: #selector(toggleOrientation), for: .touchUpInside)
-        orientLockButton.setImage(UIImage(named: "lockIcon-new"), for: .normal)
-        orientLockButton.tintColor = .orange
-        //TODO: add accessability options for orientationLockButton
-        return orientLockButton
+    lazy var interfaceLockButton: UIButton = {
+        var interfaceLockButton = UIButton(type: .system)
+        interfaceLockButton.setImage(UIImage(named: "iconLock"), for: .normal)
+        interfaceLockButton.addTarget(self, action: #selector(toggleInterfaceLock), for: .touchUpInside)
+        interfaceLockButton.tintColor = .white
+        interfaceLockButton.accessibilityHint = NSLocalizedString("INTERFACE_LOCK_HINT", comment: "")
+        interfaceLockButton.accessibilityLabel = NSLocalizedString("INTERFACE_LOCK_BUTTON", comment: "")
+        return interfaceLockButton
     }()
     
     lazy var moreOptionsButton: UIButton = {
         var moreOptionsButton = UIButton(type: .system)
+        moreOptionsButton.setImage(UIImage(named: "iconMoreOptions"), for: .normal)
         moreOptionsButton.addTarget(self, action: #selector(selectMoreOptions), for: .touchUpInside)
-        moreOptionsButton.setImage(UIImage(named: "moreWhite-new"), for: .normal)
-        moreOptionsButton.tintColor = .orange
-        //TODO: add accessability options for moreOptionsButton
+        moreOptionsButton.tintColor = .white
+        moreOptionsButton.accessibilityHint = NSLocalizedString("MORE_OPTIONS_HINT", comment: "")
+        moreOptionsButton.accessibilityLabel = NSLocalizedString("MORE_OPTIONS_BUTTON", comment: "")
         return moreOptionsButton
     }()
     
     // MARK: Class Initializers
     required init(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
+        fatalError("init(coder aDecoder: NSCoder) not implemented")
     }
     
-    override init(frame: CGRect) {
-        super.init(frame: frame)
+    private func setupView() {
         addArrangedSubview(toggleFullScreenButton)
         addArrangedSubview(selectSubtitleButton)
         addArrangedSubview(repeatButton)
-        addArrangedSubview(orientationLockButton)
+        addArrangedSubview(interfaceLockButton)
         addArrangedSubview(moreOptionsButton)
+        axis = .horizontal
+        spacing = 32
+        translatesAutoresizingMaskIntoConstraints = false
+    }
+    
+    override init(frame: CGRect) {
+        repeatMode = .doNotRepeat
+        subtitleToggled = false
+        interfaceDisabled = false
```

Comment:
we need to find a solution for this to not store this kind of information in this controller but in a general object. 
Doesn't need to be now but we need to write follow up tickets for this